benchmark_runner/common/ocp_resources/odf/template/01_delete_disks_template.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+while (($# >= 2)) ; do
+    node=$1
+    delete_disk_command=$2
```

Comment:
Should be "=$2" (in quotes) so the shell won't process the value of $2.

Reply:
Fixed